packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -300,7 +300,7 @@ export class FetchInstrumentation extends InstrumentationBase<
         const options = input instanceof Request ? input : init || {};
         const createdSpan = plugin._createSpan(url, options);
         if (!createdSpan) {
-          return original.apply(this, [url, options]);
+          return original.apply(this, arguments);
```

Comment:
I think you will need to do `as any` since we actually don't know what the user gives us (they could pass invalid input) and it could actually be anything